tests/physical_bc/apply_2d.uniThn.amrGrid.output

```diff
@@ -0,0 +1,17 @@
+|e|_oo = 8.12101
+|e|_2  = 3.07985
+|e|_1  = 2.32166
+
+Error in RHS_un :
+  L1-norm:  1.728856669
+  L2-norm:  2.921730398
+  max-norm: 8.121012094
```

Comment:
I recall discussing a bug in some implementation, but I don't remember where. Did we fix that?

Reply:
None of the programs in physical bc exhibit 2nd order convergence. I did not include any tests to check for 2nd order convergence though.

Reply:
So this is by design. The `MultiphaseStaggeredStokesOperator` uses linear extrapolation to fill ghost cell values. A 5 point Laplacian does not converge at the boundary if you use linear extrapolation. If you use quadratic extrapolation, then the operator converges.